packages/trading/src/tradingSdk.ts

```diff
@@ -161,15 +168,74 @@ export class TradingSdk {
     )
   }
 
-  async getPreSignTransaction(
-    params: WithPartialTraderParams<{ orderId: string; account: string }>,
-  ): ReturnType<typeof getPreSignTransaction> {
+  async getPreSignTransaction(params: OrderTraderParams): ReturnType<typeof getPreSignTransaction> {
     const adapter = getGlobalAdapter()
 
     const traderParams = this.mergeParams(params)
     const signer = traderParams.signer ? adapter.createSigner(traderParams.signer) : adapter.signer
 
-    return getPreSignTransaction(signer, traderParams.chainId, params.account, params.orderId)
+    return getPreSignTransaction(signer, traderParams.chainId, params.orderUid)
+  }
+
+  async getOrder(params: OrderTraderParams): Promise<EnrichedOrder> {
+    const orderBookApi = this.resolveOrderBookApi(params)
+
+    return orderBookApi.getOrder(params.orderUid)
+  }
+
+  async offChainCancelOrder(params: OrderTraderParams): Promise<boolean> {
```

Comment:
Nitpick, totally not for this PR.
Would it make sense to expose as well the method to (off-chain) cancel multiple orders? Or make this one accept multiple already?

Reply:
100% makes sense! Thanks for noticing, I will add it in another PR